Sources/SwiftKueryORM/Model.swift

```diff
@@ -0,0 +1,859 @@
+/**
+ Copyright IBM Corporation 2018
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+import SwiftKuery
+import KituraContracts
+import Foundation
+
+public protocol Model: Codable {
+  static var tableName: String {get}
+  static var idColumnName: String {get}
+
+  static func createTableSync() throws -> Bool
```

Comment:
createTableSync() and dropTableSync() should take (using: Database) as arg

Reply:
done